config/locales/devise.en.yml

```diff
@@ -7,6 +7,13 @@ en:
       send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
       send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
     failure:
+      saml_user:
+        invalid: "Log in failed. Please confirm your NetID or password."
```

Comment:
`NetID` is hard coded. But will this error ever appear? i.e. will there ever be an invalid saml_user